packages/rpc-core/src/index.ts

```diff
@@ -211,12 +213,16 @@ export default class Rpc implements RpcInterface {
       }, {} as RpcInterface[Section]);
   }
 
-  private _createMethodWithRaw (creator: (isRaw: boolean) => (...values: any[]) => Observable<any>): RpcInterfaceMethod {
-    const call = creator(false) as Partial<RpcInterfaceMethod>;
+  private _memomize (creator: (outputAs: OutputType) => (...values: any[]) => Observable<any>): RpcInterfaceMethod & memoizee.Memoized<RpcInterfaceMethod> {
+    const memoized = memoizee(creator('scale') as RpcInterfaceMethod, {
+      length: false,
```

Comment:
```suggestion
      // Dynamic length for argument	
      length: false,	
      // Normalize args so that different args that should be cached	
      // together are cached together.	
      // E.g.: `query.my.method('abc') === query.my.method(createType('AccountId', 'abc'));`
```